digitalocean-ts-loadbalanced-droplets/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "webserver-linode",
+    "version": "0.1.0",
```

Comment:
Do we need the version?

Reply:
It's on the others so I kept it there - I am happy to remove it as a clean sweep from them all if you think it would be better?

Reply:
OK, let's keep it.